packages/faustwp-core/src/getWordPressProps.tsx

```diff
@@ -69,12 +70,19 @@ export async function getWordPressProps(options: GetWordPressPropsConfig) {
 
   const seedNode = seedQueryRes?.data?.node as SeedNode;
 
+  verboseLog(`Seed Node for resolved url: "${resolvedUrl}": `, seedNode);
+
   if (!seedNode) {
     return {
       notFound: true,
     };
   }
 
+  infoLog(
```

Comment:
Explicitly calling this here so it only runs once on the server, instead of on the client too.